agents/pve/fence_pve.py

```diff
@@ -140,14 +140,23 @@ def main():
 	atexit.register(atexit_handler)
 
 	all_opt["node_name"] = {
-		"getopt" : "N:",
+		"getopt" : ":",
 		"longopt" : "nodename",
-		"help" : "-N, --nodename                 "
+		"help" : "--nodename                 "
+			"Replaced by -N, --pmx-node",
+		"required" : "0",
+		"shortdesc" : "Replaced by -N/--pmx-node",
+		"order": 2
+	}
+	all_opt["pmx_node"] = {
```

Comment:
I think it's better to put the new/sensibly named parameter first, so users see it before the backwards-comptible one.